src/definition/users/IPostUserStatusChanged.ts

```diff
@@ -0,0 +1,21 @@
+import { IHttp, IModify, IPersistence, IRead } from '../accessors';
+import { AppMethod } from '../metadata';
+import { IUserStatusContext } from './IUserStatusContext';
+
+/**
+ * Event interface that allows an app to
+ * register as a handler of the `IPostUserStatusChanged`
+ * event
+ *
+ * This event is triggered *after* the
+ * user changes his status on Rocket.chat
+ */
+export interface IPostUserStatusChanged {
+    [AppMethod.EXECUTE_POST_USER_STATUS_CHANGED](
+        context: IUserStatusContext,
```

Comment:
Will this include the custom status? And the indicators, whether `busy`, `away`, `online`, etc?

Reply:
At the moment, we will not add the custom statuses, only `online`, `offline`, `busy`, and `away`.